app/controllers/backoffice/avis_controller.rb

```diff
@@ -67,6 +68,20 @@ def update_params
     params.require(:avis).permit(:answer)
   end
 
+  def redirect_if_no_sign_up_needed
+    avis = Avis.find(params[:id])
+
+    # a gestionnaire is authenticated ... lets see if it can view the dossier
+    if current_gestionnaire.present?
```

Comment:
Oui pour le return visiblement. http://stackoverflow.com/questions/5743534/redirect-to-return